packages/universal-xml-plugin/lib/xpath.js

```diff
@@ -1,10 +1,19 @@
 import {select as xpathQuery} from 'xpath';
-import {DOMParser, MIME_TYPE} from '@xmldom/xmldom';
+import {DOMParser, MIME_TYPE, Document} from '@xmldom/xmldom';
 import _ from 'lodash';
 
+/**
+ * @param {Document} doc - The Document object to be converted.
+ * @returns {Node} - The converted Node object.
+ */
+function convertDocumentToNode(doc) {
```

Comment:
Do we need a separate function for such conversion?

Reply:
I think no, this was middle of modifications to see what was happening